products/intelprocessors.md

```diff
@@ -0,0 +1,171 @@
+---
+permalink: /intel-processors
+title: Intel Processors
+iconSlug: intel
+category: device
+versionCommand: "'msinfo32' on Windows, 'cat /proc/cpuinfo' on Linux"
```

Comment:
`cpuinfo` doesn't return my processor model for me (`skylake`). `cat /sys/devices/cpu/caps/pmu_name` does work.

Reply:
Changed with https://github.com/endoflife-date/endoflife.date/pull/1397/commits/9f303fee3f958a4646cbe62839b705fe0793d4bb